full/src/test/java/apoc/load/LoadHtmlTestParameterized.java

```diff
@@ -101,13 +103,13 @@ public void testQueryH2WithConfig() {
         Map<String, Object> config = map(confList.toArray());
 
         skipIfBrowserNotPresentOrCompatible(() -> {
-            testResult(db, "CALL apoc.load.html($url, $query, $config)",
+            assertEventually(() -> db.executeTransactionally("CALL apoc.load.html($url, $query, $config)",
                     map("url",new File("src/test/resources/wikipedia.html").toURI().toString(), "query", query, "config", config),
                     result -> {
                         Map<String, Object> row = result.next();
                         assertEquals(map("h2",asList(RESULT_QUERY_H2)).toString().trim(), row.get("value").toString().trim());
-                        assertFalse(result.hasNext());
-                    });
+                        return !result.hasNext();
+                    }, Duration.ofSeconds(3)), (v) -> v, 30, TimeUnit.SECONDS);
```

Comment:
This retries on all errors but maybe we just want to retry on TimeoutExceptions?

Reply:
Yes, makes sense. Just changed, and now the query is re-executed only after a transaction timeout.

Also, even though the error seems present only in `testQueryH2WithConfig`, it could potentially occur for other selenium tests as well. So I added retry in all these tests.